README.md

```diff
@@ -1,14 +1,66 @@
+
+Categorizing features, contacts saved in the Contact Manager, can be grouped/labeled for better organizing purposes. Users can use this software to do a quick search for a group of people who share the same information, e.g from the same company.
+
+## Setting up in Intellij
+
+Prerequisites: JDK 11 (use the exact version), update Intellij to the most recent version.
+
+1. **Ensure Intellij JDK 11 is defined as an SDK**, as described [here](https://www.jetbrains.com/help/idea/sdk.html#set-up-jdk) -- this step is not needed if you have used JDK 11 in a previous Intellij project.
+2. **Import the project _as a Gradle project_**, as described [here](https://se-education.org/guides/tutorials/intellijImportGradleProject.html).
+3. **Verify the set up**: After the importing is complete, locate the `src/main/java/seedu/duke/Duke.java` file, right-click it, and choose `Run Duke.main()`. If the setup is correct, you should see something like the below:
+   ```
+   > Task :compileJava
+   > Task :processResources NO-SOURCE
+   > Task :classes
+   
+   > Task :Duke.main()
+   Hello from
+    ____        _        
+   |  _ \ _   _| | _____ 
+   | | | | | | | |/ / _ \
+   | |_| | |_| |   <  __/
+   |____/ \__,_|_|\_\___|
+   
+   What is your name?
+   ```
+   Type some word and press enter to let the execution proceed to the end.
+
+
+## Testing
+
+### I/O redirection tests
+
+* To run _I/O redirection_ tests (aka _Text UI tests_), navigate to the `text-ui-test` and run the `runtest(.bat/.sh)` script.
+
+### JUnit tests
+
+* A skeleton JUnit test (`src/test/java/seedu/duke/DukeTest.java`) is provided with this project template.
+* If you are new to JUnit, refer to the [JUnit Tutorial at se-education.org/guides](https://se-education.org/guides/tutorials/junit.html).
+
+## Checkstyle
+
+* A sample CheckStyle rule configuration is provided in this project.
+* If you are new to Checkstyle, refer to the [Checkstyle Tutorial at se-education.org/guides](https://se-education.org/guides/tutorials/checkstyle.html).
+
+## CI using GitHub Actions
+
+The project uses [GitHub actions](https://github.com/features/actions) for CI. When you push a commit to this repo or PR against it, GitHub actions will run automatically to build and verify the code as updated by the commit/PR.
+
+
+## Documentation
+
+`/docs` folder contains a skeleton version of the project documentation.
+
+Steps for publishing documentation to the public:
+1. If you are using this project template for an individual project, go your fork on GitHub.<br>
+   If you are using this project template for a team project, go to the team fork on GitHub.
+1. Click on the `settings` tab.
+1. Scroll down to the `GitHub Pages` section.
+1. Set the `source` as `master branch /docs folder`.
+1. Optionally, use the `choose a theme` button to choose a theme for your documentation.
```

Comment:
As per [instructions](https://nus-tic4002-ay2122s2.github.io/website/schedule/week6/project.html), we need to acknowledge original source